docs/develop/dotnet/redis-om-dotnet/connecting-to-redis/connecting-to-redis.md

```diff
@@ -0,0 +1,65 @@
+---
+id: connecting
+title: Connecting to Redis with Redis OM Dotnet
+sidebar_label: Connecting to Redis
+slug: /develop/dotnet/redis-om-dotnet/connecting-to-redis
+---
+
+Redis OM is designed to make using Redis easier for .NET developers, so naturally the first question one might ask is how would you use it to connect to Redis?
+
+The Redis OM library is an abstraction above a lower level (closer to Redis) library—[StackExchange.Redis](https://github.com/StackExchange/StackExchange.Redis)—which it uses to manage connections to Redis. That is however, an implementation detail which should not be a concern to the user. `RedisConnectionProvider` class contains the connection logic, and provides for connections to Redis. The RedisConnectionProvider should only be initialized once in your app's lifetime.
+
+## Initializing RedisConnectionProvider
+
+RedisConnectionProvider takes a [Redis URI](https://github.com/redis-developer/Redis-Developer-URI-Spec/blob/main/spec.md) and uses that to initialize a connection to Redis.
+
+Consequentially, all that needs to be done to initialize the client is calling the constructor of `RedisConnectionProvider` with a Redis uri. Alternatively, you can connect with a ConnectionConfiguration object.
+
+### Connecting to a Standalone Instance of Redis No Auth
```

Comment:
Unfortunately the formatting on Redis Developer makes these H3 too big. Maybe make the H4s or even H5s?